pkg/platform/aws/template.go

```diff
@@ -118,7 +118,7 @@ module "worker-pool-{{ $index }}" {
   cluster_domain_suffix = "{{ $.Config.ClusterDomainSuffix }}"
   {{- end }}
 
-  ssh_keys              = {{ index (index $.WorkerpoolCfg $index) "ssh_pub_keys" }}
+  ssh_keys              = {{ (index $.WorkerpoolCfg $index "ssh_pub_keys") }}
```

Comment:
Why not use `$pool.SSHPubKeys`. I mean:
```suggestion
  ssh_keys              = {{ $pool.SSHPubKeys }}
```

Reply:
This can't work because that is a slice, we woud need like:
```suggestion
    ssh_keys = [
      {{- range $ke := $pool.SSHPubKeys }}
      {{ $key }} = "{{ $value }}"
      {{- end }}
    ]
```